Add a live demo for how to use proj4.js in case of Highmaps charts/maps.

### Requested feature description
Add a live demo for how to use proj4.js in case of Highmaps charts/maps.

@KacperMadej still not working with `proj4` as expected.

Steps to replicate:

1. Clone repository, install and [run the demo](https://github.com/highcharts/highcharts-angular#demo-app) - this works successfully
2. `npm i proj4 --save`
3. Copy `app.module.ts`, `app.component.ts`, and `app.component.html` code from the [codesandbox demo](https://codesandbox.io/s/angular-0b1w7)
4. `npm run start` will throw a few compilation errors and fail to compile.
    - `ERROR in src/app/app.component.ts(18,11): error TS2345: Argument of type 'typeof import("C:/Users/Dev/Documents/highcharts-angular/node_modules/highcharts/highcharts.src")' is not assignable to parameter of type 'typeof import("C:/Users/Dev/Documents/highcharts-angular/node_modules/highcharts/highcharts")'.`
    - `Property 'geojson' is missing in type 'typeof import("C:/Users/Dev/Documents/highcharts-angular/node_modules/highcharts/highcharts.src")'.`
    - `src/app/app.component.ts(21,13): error TS2339: Property 'win' does not exist on type 'typeof import("C:/Users/Dev/Documents/highcharts-angular/node_modules/highcharts/highcharts.src")'.`
![image](https://user-images.githubusercontent.com/3241448/72850942-66117000-3cfe-11ea-9689-1dd45e26bfc2.png)
5. Save any file again and it will compile successfully. E.g. open `app.component.ts` and press `Ctrl+S` to save and trigger a compilation. The errors will still be logged in the terminal but this time it compiles successfully.
6. Go to http://localhost:4200/ and the map no longer loads; errors are logged to the console relating to `proj4` and http://www.highcharts.com/errors/22
![image](https://user-images.githubusercontent.com/3241448/72851352-83930980-3cff-11ea-989c-6fef5c209578.png)

To make the above replication easier, I've forked the demo, made my changes and published to https://github.com/ashnewport/highcharts-angular. You can clone that if you'd like and execute `npm start` to save some time and a few steps from above.

Some additional tweaks which I have _not_ committed to my fork:
I can get rid of the compilation errors with a few castings which doesn't feel super great but inside `app.component.ts` update to:
```typescript
// init map module
MapModule(Highcharts as any);

// add proj4 to window, so Highcharts will be able to find it
(Highcharts as any).win = (Highcharts as any).win || {};
(Highcharts as any).win.proj4 = proj4;
```
Just for extra measure `npm install highcharts-angular --save` so we can visually see it in the `package.json` as `"highcharts-angular": "file:highcharts-angular"` when using the demo app.

I encounter the same issues when installing with the latest Angular, Highcharts and proj4 libraries so I don't know if this is a version compatibility issue or just some missing interfaces.

Lastly, if you feel this is getting off topic from the GitHub Issue then I'm happy to move over to the forms for further support.

--- a/README.md
+++ b/README.md
@@ -375,6 +375,7 @@ Contains the chart component that creates Highcharts chart.
 * Stock + indicators: [https://stackblitz.com/edit/highcharts-angular-stock-indicators](https://stackblitz.com/edit/highcharts-angular-stock-indicators)
 * Stock + GUI: [https://stackblitz.com/edit/highcharts-angular-stock-gui](https://stackblitz.com/edit/highcharts-angular-stock-gui)
 * Map: [https://stackblitz.com/edit/highcharts-angular-map](https://stackblitz.com/edit/highcharts-angular-map)
+* Map + proj4: [https://stackblitz.com/edit/highcharts-angular-map-proj4](https://stackblitz.com/edit/highcharts-angular-map-proj4)
 * Optimal way to update: [https://stackblitz.com/edit/highcharts-angular-optimal-way-to-update](https://stackblitz.com/edit/highcharts-angular-optimal-way-to-update)
 * Data from the service: [https://stackblitz.com/edit/highcharts-angular-data-from-service-2](https://stackblitz.com/edit/highcharts-angular-data-from-service-2)
 * Applying a custom plugin/wrap: [https://stackblitz.com/edit/highcharts-angular-custom-plugin](https://stackblitz.com/edit/highcharts-angular-custom-plugin)
@@ -430,3 +431,7 @@ The correct repository to report such issues is [main Highcharts repository](htt
 
 Based on original Highcharts demo for [Synchronized charts](https://www.highcharts.com/demo/synchronized-charts).
 Additionally added class based sync between charts - [demo](https://codesandbox.io/s/5wwz8qy1l4).
+
+#### How to use Highcharts Maps with the proj4?
+
+Install the `proj4` library and its types `@types/proj4`. Then pass it to `chartOptions.chart.proj4` property. See the [demo app](#demo-app) in this repository or [live demo](https://stackblitz.com/edit/highcharts-angular-map-proj4) example.

--- a/README.md
+++ b/README.md
@@ -437,8 +437,6 @@ Additionally added class based sync between charts - [demo](https://codesandbox.
 
 Install the `proj4` library and its types `@types/proj4`. Then pass it to `chartOptions.chart.proj4` property. See the [demo app](#demo-app) in this repository or [live demo](https://stackblitz.com/edit/highcharts-angular-map-proj4) example.
 
-Additionally, added class based sync between charts - [demo](https://codesandbox.io/s/5wwz8qy1l4).
-
 #### I want to render angular component in the tooltip/axis formatter
 
 To render angular component within the chart you can use the angular [portals](https://material.angular.io/cdk/portal/overview) - [demo](https://stackblitz.com/edit/highcharts-angular-portal-usage)

